p2p/host.go

```diff
@@ -56,6 +56,8 @@ const (
 	SetAsideOtherwise = 1 << 12
 	// MaxMessageHandlers ..
 	MaxMessageHandlers = SetAsideForConsensus + SetAsideOtherwise
+	// MaxMessageSize is the 256Kb
+	MaxMessageSize = 1 << 18
```

Comment:
Our block can be as big as around 2MB.

Reply:
Good catch, then we need to adjust it.  The current default max is actually 1Mb.